# HW-23: Due Friday 21 July 2017 by 2:00:00 pm EDT 

A Rankine cycle power plant has a pump whose isentropic efficiency is 75%  and a turbine whose isentropic efficiency is 97%.  The pump receives saturated liquid water at 0.1 bar.  The boiler outlet stream is as 40 bar and 400 C. 

Compute the cycle energy efficiency and compare it one whose pump and turbine are isentropic.

# Problem:

Compute the cycle energy efficiency and compare it one whose pump and turbine are isentropic.

# Knowns:

- Cycle: Rankine
- isentropic efficiencies: 
  - pump: 75% 
  - turbine: 97%
- Pump In:
  - Saturated Liquid Water
  - Pressure: 0.1 bar
- Boiler Output:
  - Steam
  - 40 bar
  - 400 C
  
# Assumptions:
- Rotating machinery is adiabatic.
- Steady flow steady state conditions (SFSS)
- 
  
# System

![](23_sys.png)

# Solution

In [22]:
from pint import UnitRegistry
ureg = UnitRegistry()
_bar = ureg.bar
_MPa = ureg.MPa

In [28]:
import sys
import os
sys.path.append(os.path.abspath("venv/lib/python3.5/site-packages/"))

from iapws import IAPWS95

In [29]:
pump_in=IAPWS95(P=0.01, x=0) #saturated water
print(pump_in.h)

191.805944621


- $$h_1 = 192 \frac{kJ}{kg}$$
- $$h_2 = 196 \frac{kJ}{kg}$$
- $$h_3 = 3210 \frac{kJ}{kg}$$
- $$h_4 = 2140 \frac{kJ}{kg}$$